### PR TITLE
Integrated PSAFeedback to dst-selector

### PR DESCRIPTION
**Pull request by nisarg210**
Replace the Feedback Page with the newly created PSAFeedback Component from shared-react-component

### CODE REVIEW

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://,172.171.52.80:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://172.171.52.80:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

File: package.json [hunk 1]
```diff
@@ -16,7 +16,9 @@
     "@psa/dst.ui.map": "^0.0.61",
     "@turf/area": "^6.5.0",
     "@turf/centroid": "^6.5.0",
+    "@vitejs/plugin-react": "^4.3.1",
     "dayjs": "^1.11.8",
+    "decision-support-tool": "file:",
```
Comment (YazidZidane) [inline package.json:21]: Remove this if not needed

**Review by YazidZidane — COMMENTED** [review]
Can you also remove the previous feedback component under `src/pages/Feedback` ?

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://172.171.52.80:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://172.171.52.80:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Review by YazidZidane — APPROVED** [review]

**Pull request merged**